.changeset/pink-ligers-share.md

```diff
@@ -0,0 +1,51 @@
+---
+"astro": minor
+---
+
+Add experimental rewriting in Astro, via `rewrite()` function and `next()` function.
+
+The feature is available via experimental flag:
+
+```js
+export default defineConfig({
+  experimental: {
+    rewriting: true
+  }
+})
+```
+
+When enabled, you can use `rewrite()` to **render
```

Comment:
Just checking on this formatting here. Did you mean to split up into two lines like this?